### ISSUE TITLE
Database Save, database restore, database export

### ISSUE THREAD

**Issue by Apaxe** [Templarian/MaterialDesign#1304]

**Comment by Templarian** [comment]
We're in the habit of not combining two icons. So for these actions it might be ideal to use the singular icons for `content-save`, `restore`, and `export`.

With that said I think having a `database-import` (arrow left side right) / `database-export` (arrow right side pointing right) would be very useful.

**Comment by GreenTurtwig** [comment]
Originally had the arrows 2px inwards, but moved them out as they cut off too much from the database icon.
I don't mind `database-import`, but I'm not sure how I feel about `database-export`. :cat2:

[![](https://user-images.githubusercontent.com/15905797/34074585-4c0ec924-e2aa-11e7-8d03-f5c0454f789a.png)](https://github.com/Templarian/MaterialDesign/files/1565113/database-import.zip)[![](https://user-images.githubusercontent.com/15905797/34074590-51d514ee-e2aa-11e7-96ac-7ac41b1c83e8.png)](https://github.com/Templarian/MaterialDesign/files/1565112/database-export.zip)

**Comment by sitecarp** [comment]
I found these icons in the search, but am unable to use them in the actual code, even through the CDN version. I assume it's because they're not actually released yet.

Additionally, I found the pending icons counter-intuitive. The arrows for both icons point _away_ from the DB icon, which to me indicated "export" in both cases. So then I assumed the one pointing diagonally up/right must mean "import" since "up" is generally used for "upload to server." Wrong. The icon with the arrow pointing _down_ and _away_ from the DB icon is the one that means "import." Why? Is it too late to revise these to have arrows pointing in (import) and out (export) of the DB icon?
